### ISSUE TITLE
can't skip (get an answer) to a script I am stuck on

### ISSUE THREAD

**Issue by rbw1234** [swirldev/swirl#288]
Hi. I am in lesson 9 (Functions) and am stuck on a script (mad_libs) that I cannot seem to write correctly. skip() does not work on scripts. How can I move forward?? Thanks.

**Comment by seankross** [comment]
You can find answers to the function lesson scripts [here](https://gist.github.com/seankross/912fe3b9f19a91b0a7bd).

**Comment by rbw1234** [comment]
Thank you!!! That helps me move forward.

**Comment by rbw1234** [comment]
Now I am stuck on the "Good" %p% "job!" exercise but there is no answer to that in the supplied link.

Why does swirl not have a way to get answers to scripts? I might not be able to finish a given chapter if I get stuck. Thanks.

**Comment by seankross** [comment]
The first function from the link above should help you out.

**Comment by SusanneKnoll** [comment]
Dear Seankross, 

Thank you for your help....first I thought that I had also a mistake in the code....but than I saw your code....and I figured that mine was totally fine. But still, swirl does not let me advance, nor skip the task. So I really do not understand what the matter is. 
See below the prints screen with my situation. 
![swirl problem](https://cloud.githubusercontent.com/assets/11787984/7672460/3a35b05a-fcca-11e4-8845-becd36e0826c.jpg)
